core/arch/arm/plat-hikey/spi_test.c

```diff
@@ -33,10 +33,18 @@
 #include <trace.h>
 #include <util.h>
 
-#define PL022_STAT		0x00C
+#define PL022_STAT	0x00C
 #define PL022_STAT_BSY	SHIFT_U32(1, 4)
 
-static __maybe_unused void spi_cs_callback(enum gpio_level value)
+static void spi_some_delay(size_t mult)
+{
+	size_t i;
+
+	for (i = 0; i < (mult*10000000); i++)
+		;
+}
```

Comment:
You can't rely on this to provide a "sleep at least this long" semantic. You should probably use `void tee_time_wait(uint32_t milliseconds_delay)`, which sleeps in normal world, or if it doesn't work for some reason then implement `void tee_time_mdelay(uint32_t ms)` which would actively wait but use `tee_time_get_sys_time()` to determine the exit condition rather than a hardcoded loop count.